content/docs/release-notes/2305-release-notes.md

```diff
@@ -0,0 +1,187 @@
+---
+linktitle: 23.05.0 Release Notes
+title: Release Notes for Open 3D Engine 23.05.0
+description: Full release notes for Open 3D Engine (O3DE) version 23.05.0.
+weight: 895
+toc: true
+---
+
+## sig-build
+
+* Test Impact Analysis Framework rollout ([#10660](https://github.com/o3de/o3de/issues/10660))
+	* Test Impact Analysis Framework (TIAF) is a system that determines which tests need to run for every source change. The idea behind is to only run the tests that you need and that your change may impact. The system is now deployed for both Python and native C++ tests. Developers working on O3DE should on average see a reduction in the time they have to wait for their tests to run.
+
+## sig-content
+
+* Adds support for global python scripts per asset type to simply customizations in the scene pipeline.
+* A new Asset Browser experience has been prepared and turned on by default. The new Asset Browser experience includes three new layout modes (table view, thumbnail view, list view), file operations (add, delete, edit, rename, move), asset drag-and-drop support, URL bar for easy sharing and editing, breadcrumb and navigation bar, new detached Asset Browser inspector panel, filter options to hide unusable Editor or engine assets, and all new asset icons.
+	* To disable the new Asset Browser, disable the `ed_useWIPAssetBrowserDesign` console variable from the **Tools** menu.
+	* If the new Asset Browser layout is not properly set, restore the default layout from the **View** -> **Layouts** menu option.
+	* Searching is disabled in the table view while this feature is being developed, please use the thumbnail or list view to search for assets.
+	* Check out all the work done in the [Asset Browser project](https://github.com/orgs/o3de/projects/16) page!
+* Gems creators can now specify compatible platforms, which can be filtered in the Gem catalog.
+* Users can choose to edit existing gems `.json` files via the **Create a Gem** wizard or CLI.
+* O3DE utility widgets in the AzQtComponents library are now exposed to Python and PySide. This allows Python-based developers of tools and gems to use or extend AzQtComponents for tools, plugins, or automation.
+* Provides the ability to manage and visualize prefab overrides from O3DE Editor's Entity Outliner. Users can now override prefabs by adding or removing entities, nested prefabs, and components or changing component properties. For more information, please refer to the [Override a Prefab](https://www.o3de.org/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab/) page.
```

Comment:
because some of these links won't be live, if there's not already, there should be a check post-release to make sure all links are working